src/pages/instances/InstanceList.tsx

```diff
@@ -126,22 +120,33 @@ const InstanceList: FC = () => {
   };
 
   const filteredInstances = instances.filter((item) => {
-    if (query) {
-      const q = query.toLowerCase();
-      if (
-        !item.name.toLowerCase().includes(q) &&
-        !item.status.toLowerCase().includes(q) &&
-        !item.type.toLowerCase().includes(q) &&
-        !item.description.toLowerCase().includes(q) &&
-        !item.config["image.description"]?.toLowerCase().includes(q)
-      ) {
-        return false;
-      }
+    if (
+      filters.queries.length > 0 &&
+      !filters.queries.every((q) => item.name.toLowerCase().includes(q)) &&
```

Comment:
The logic is a bit off. For every query we want it to match at least one of name, descripiton or image I think like this:
```
!filters.queries.every((q) => 
  item.name.toLowerCase().includes(q) || 
  item.description.toLowerCase().includes(q) ||
  item.config["image.description"]?.toLowerCase().includes(q)
)
```

Reply:
I think we can remove the `filters.queries.length > 0` because if is an empty array the `every` is implicitly true.